src/Router.php

```diff
@@ -17,14 +17,71 @@ public static function getInstance()
         return static::$instance;
     }
     
+    /**
+     * Loads the location of the routes file.
+     * 
+     * @return string
+     */
+    protected function getRouteFileLocation()
+    {
+        $dir = getenv('HOME') . '/.volcano-sdk';
+        
+        if (!is_dir($dir)) {
+            mkdir($dir);
+        }
+        
+        return $dir . '/routes.json';
+    }
+    
+    /**
+     * Loads Routes
+     */
     public function loadRoutes()
     {
+        $rfile = $this->getRouteFileLocation();
+        
+        if (!file_exists($rfile)) {
+            $this->loadRoutesFromService();
+            return;
+        }
+        
         $this->_routeList = json_decode(
-            file_get_contents(dirname(__FILE__) . '/../routes.json'),
+            file_get_contents($rfile),
             true
         );
     }
     
+    /**
+     * Loads the routs fromm the api endpoint and stores them locally.
+     */
+    public function loadRoutesFromService()
+    {
+        $service = new Service('endpoints');
```

Comment:
Yep, uses the same API Key which is previously defined. It basically just makes a virtual service.